index.html

```diff
@@ -40,5 +40,14 @@
     <mgt-agenda group-by-day></mgt-agenda>
     <mgt-people show-presence></mgt-people>
     <mgt-todo></mgt-todo>
+    <mgt-get resource="me">
```

Comment:
Awesome. Could you move this to a story in storybook here: https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/main/stories/components/get.stories.js ?

Once you've added a new story in that file, you can `yarn storybook:dev` and then `yarn storybook` to launch storybook locally and test.

Reply:
Makes a lot of sense!